app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,43 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = Post.where(author_id: params[:user_id]).order(created_at: :desc).limit(2)
+  end
+
+  def show
+    @user = User.find(params[:user_id])
+    @post = Post.where(author_id: params[:user_id]).find(params[:id])
+    comment = Comment.new
```

Comment:
**[OPTIONAL]** 

- [ ] Inside your post controller, I would like to recommend you implement **params** method you can find more details about it right here :+1:
```ruby
   private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```